src/main/scala/com/exasol/cloudetl/kinesis/KinesisShardDataImporter.scala

```diff
@@ -20,23 +20,19 @@ object KinesisShardDataImporter {
     val region = kinesisArgumentsReader.getRegionArgument
     val shardId = kinesisArgumentsReader.getShardIdArgument
     val shardSequenceNumber = kinesisArgumentsReader.getShardSequenceNumber
```

Comment:
Same here, create, `KinesisConsumerProperties`, using the first string from iterator.

Reply:
done